staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -53,7 +54,8 @@ type AuditOptions struct {
 	PolicyFile string
 
 	// Plugin options
-
+	// Defaults to false.
+	EnableSyncMode bool
```

Comment:
I have strange question.

If I want to use unbuffered mode in log backend, but use buffered mode in webhook mode, what should I do?

Reply:
This is quite a strange requirements, I guess no one will need this.
I will only open one backend at a time.

Reply:
@crassirostris Should we reserve to original options?

Reply:
I think this is not a strange requirement: buffered webhook and synchronous log

Additionally, there are several options at the buffering stage and we will at some point make them configurable. Why would you not do as I suggested earlier, moving buffering options to both LogOptions and WebhookOptions?

Reply:
ok